frontend/components/__tests__/TodayInHistory.test.tsx

```diff
@@ -27,85 +27,38 @@ describe("TodayInHistory", () => {
   };
 
   it("renders", () => {
-    render(<TodayInHistory />);
-  });
-
-  it("starts in loading state", () => {
-    const { queryByText } = render(<TodayInHistory />);
-    expect(queryByText("Fetching content")).toBeInTheDocument();
+    render(<TodayInHistory modules={[]} />);
   });
 
   describe("with content", () => {
-    const serverWithContent = setupServer(
-      rest.get("/api/home/today_in_history/", (req, res, ctx) => {
-        return res(ctx.json([testModule]));
-      })
-    );
-
-    beforeAll(() => {
-      serverWithContent.listen();
-    });
-
-    afterAll(() => {
-      serverWithContent.close();
-    });
-
     it("renders modules", async () => {
-      const { findByText } = render(<TodayInHistory />);
+      const { findByText } = render(<TodayInHistory modules={[testModule]} />);
       expect(await findByText(testModule.title)).toBeInTheDocument();
     });
 
     it("has modules linked to details pages", async () => {
       mockRouter.setCurrentUrl("/");
-      const { findByText } = render(<TodayInHistory />);
+      const { findByText } = render(<TodayInHistory modules={[testModule]} />);
       userEvent.click(await findByText(testModule.title));
       expect(Router.asPath).toStrictEqual(testModule.absoluteUrl);
     });
   });
 
   describe("without content", () => {
-    const serverWithoutContent = setupServer(
-      rest.get("/api/home/today_in_history/", (req, res, ctx) => {
-        return res(ctx.json([]));
-      })
-    );
-
-    beforeAll(() => {
-      serverWithoutContent.listen();
-    });
-
-    afterAll(() => {
-      serverWithoutContent.close();
-    });
-
     it("shows 'no modules' message", async () => {
-      const { findByText } = render(<TodayInHistory />);
+      const { findByText } = render(<TodayInHistory modules={[]} />);
       expect(
         await findByText("There are no modules associated with this date.")
       ).toBeInTheDocument();
     });
   });
 
-  describe("with error response", () => {
-    const serverWithError = setupServer(
-      rest.get("/api/home/today_in_history/", (req, res, ctx) => {
-        return res(ctx.status(500), ctx.json({ errorMessage: "test error" }));
-      })
-    );
-
-    beforeAll(() => {
-      serverWithError.listen();
-    });
-
-    afterAll(() => {
-      serverWithError.close();
-    });
-
-    it("shows 'no modules' message", async () => {
-      const { findByText } = render(<TodayInHistory />);
-      expect(
-        await findByText("There are no modules associated with this date.")
-      ).toBeInTheDocument();
+  describe("module card clicked", () => {
+    it("renders clicked module card", async () => {
+      mockRouter.setCurrentUrl("/");
+      const { findByText } = render(<TodayInHistory modules={[testModule]} />);
+      userEvent.click(await findByText(testModule.title));
+      expect(Router.asPath).toStrictEqual(testModule.absoluteUrl);
```

Comment:
This test looks the exact same as https://github.com/ModularHistory/modularhistory/blob/07eb6d7a1c2cb9735622ce30e396359e713f75fb/frontend/components/__tests__/TodayInHistory.test.tsx#L39-L44

I think we don't need it.

Reply:
Removed it!